CHANGELOG.adoc

```diff
@@ -29,6 +29,8 @@ A release with an intentional breaking changes is marked with:
 ** {issue}642[#642]: Add `driver-type` to retrieve driver type keyword. ({person}dgr[@dgr])
 ** {issue}644[#644]: Deprecate `when-predicate` and `when-not-predicate` macros. See docstrings for guidance on alternatives. These will may be removed from the API at the next release that contains breaking changes. ({person}dgr[@dgr])
 ** {issue}647[#647]: Fix logic bug in `intersects?`. Added tests to test suite. ({person}dgr[@dgr])
+** {issue}633[#633]: The `dispatch-driver` function is an implementation function and has been marked `:private`.
+** {issue}633[#633]: Duplicate definitions of `locator-xpath` and `locator-css` have been removed from the `etaoin.api` namespace. If you need these definitions, use the ones in `etaoin.query`.
```

Comment:
Can you reword this? This sounds like a breaking change, but is not.
These vars were not part of the public API in the api ns, so folks should not have been using them anyway.
Actually... since it is non-breaking, you can just not mention it in the changelog.

Reply:
So, you just want me to strip it out entirely? If so, can do. Easy peasy.

Reply:
OK, done.